src/components/organisms/NavBarSchedule/NavBarSchedule.tsx

```diff
@@ -57,6 +59,7 @@ interface UserWithVenueIdProps extends WithId<User> {
 
 export const NavBarSchedule: React.FC<NavBarScheduleProps> = ({
   isVisible,
+  closeEventSchedule = () => {},
```

Comment:
This prop should be called `onClose` or `onClickClose` That way it indicates what the callback will be used for without you having to go inside the component and look at the logic.

Callback props are usually named with the `on` prefix. Ex: `onChange` `onHide` `onClick` and etc...

Reply:
fixed in 8d545e3dc7eb572719af9ed540543a75030c9908